okhttp/src/main/kotlin/okhttp3/OkHttpClient.kt

```diff
@@ -77,9 +77,19 @@ import org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement
  *
  * You can customize a shared OkHttpClient instance with [newBuilder]. This builds a client that
  * shares the same connection pool, thread pools, and configuration. Use the builder methods to
- * configure the derived client for a specific purpose.
+ * add configuration to the derived client for a specific purpose.
  *
- * This example shows a call with a short 500 millisecond timeout:
+ * This example shows the single instance with default configurations.
+ * 
+ * ```
+ * public final OkHttpClient client = new OkHttpClient.Builder()
+ *     .readTimeout(1000, TimeUnit.MILLISECONDS)
+ *     .writeTimeout(1000, TimeUnit.MILLISECONDS)
+ *     .build();
+ * ```
+ *
+ * This example shows a call with a short 500 millisecond read timeout and a 1000 millisecond 
```

Comment:
doesn't match above. 500 vs 1000.

Reply:
This phrase is for the example at bottom. 1000 write timeout inherited from the original and 500 overridden with the new builder configuration. Isn't it ? What's your point ? @yschimke

Reply:
I read in the context of the one above.  makes sense.